### PR TITLE
Fixes for API calls, authorization and model importing

### PR DESCRIPTION
**Pull request by dandruszak-asi**

### CODE REVIEW

**Review by wswin5k — APPROVED** [review]

File: mlflow_export_import/client/http_client.py [hunk 1]
```diff
@@ -233,6 +233,18 @@ class MlflowHttpClient(HttpClient):
...
+        rsp = requests.get(uri, headers=self._mk_headers(), params=params, timeout=_TIMEOUT)
+        checked_response = self._check_response(rsp, params)
+        return self._json_loads(checked_response, params)
+
+    def _mk_headers(self):
+        headers = { "User-Agent": USER_AGENT, "Content-Type": "application/json" }
+        if self.token:
+            headers["Authorization"] = f"Basic {self.token}"
+        return headers
+
```
Comment (wswin5k) [inline mlflow_export_import/client/http_client.py:247]: this seems to be just copied from the `HttpClient` base class 🤔

Comment (dandruszak-asi) [reply mlflow_export_import/client/http_client.py:247]: it is, but not 1:1 - the autorization header content is different - it's `Bearer` in the base class, replaced with `Basic` in the subclass

Comment (wswin5k) [reply mlflow_export_import/client/http_client.py:247]: oh I see, so the get method could be the same I guess

Comment (dandruszak-asi) [reply mlflow_export_import/client/http_client.py:247]: the devil's in the details :D the original `get` method passes `data=params` to the http request, the overwritten one passes `params=params`. The former doesn't work with MLFlow's api, the latter does :) But yeah, it's true these are very minimal changes

Comment (wswin5k) [reply mlflow_export_import/client/http_client.py:247]: ok, sory I'm blind

Comment (dandruszak-asi) [reply mlflow_export_import/client/http_client.py:247]: no worries :)

**Pull request merged**